bin/interpreter.sh

```diff
@@ -204,10 +204,13 @@ elif [[ "${INTERPRETER_ID}" == "pig" ]]; then
   fi
 elif [[ "${INTERPRETER_ID}" == "flink" ]]; then
```

Comment:
#discussing Maybe we should refactor this into a separate functions loaded from an interpreter-specific scripts? It could be easier to understand the code of interpreter as we add more and more interpreters. Plus, this will make it easier to have external interpreters.

Reply:
I thought about it before, but haven't decided how to do that.

Reply:
I'll put it into my list of TODOs, to think about it...

Reply:
Thanks @alexott  I believe we could do it in 0.10